utilities/create-mock-data/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using Prime;
+using Prime.Models;
+using PrimeTests.ModelFactories;
+using Serilog;
+
+namespace create_mock_data
+{
+    class Program
+    {
+        /// <summary>
+        /// See the README.md file
+        /// </summary>
+        static void Main(string[] args)
+        {
+            Log.Logger = new LoggerConfiguration()
+                .WriteTo.Console()
+                .CreateLogger();
+
+            if (args.Length == 0) 
+            {
+                Log.Warning("Usage:  `dotnet run <number of Enrollees to generate> <number of Sites to generate>`");
+                return;
+            }
+
+            ApiDbContext dbContext = new ApiDbContextFactory().CreateDbContext(args);
+            int numEnrollees = int.Parse(args[0]);
+            if (numEnrollees > 0)
+            {
+                try
+                {
+                    Log.Information($"Beginning to generate {numEnrollees} enrollees at {DateTime.Now}");
+                    for (int i = 0; i < numEnrollees; i++)
+                    {
+                        Enrollee enrollee = new EnrolleeFactory().Generate();
```

Comment:
You should be able to make one factory and call `Generate()` or `Generate(x_` on it rather than make a new one for each Enrollee you want to make, This should help performance.